src/ReservoirPriceOracle.sol

```diff
@@ -504,9 +513,12 @@ contract ReservoirPriceOracle is IPriceOracle, IReservoirPriceOracle, Owned(msg.
         emit RewardGasAmount(aNewMultiplier);
     }
 
-    // sets a specific pair to serve as price feed for a certain route
+    /// @notice Sets the pair to serve as price feed for a given route.
+    // TODO: Should be TokenA & TokenB because this function sorts them?
     function designatePair(address aToken0, address aToken1, ReservoirPair aPair) external onlyOwner {
         (aToken0, aToken1) = aToken0.sortTokens(aToken1);
+        // TODO: Should be require as it's possible to fail right? Generally
```

Comment:
the only way it can fail is from the incompetence of the admin. I was debating whether to make the func sig to just `designatePair(ReservoirPair aPair)` and then it just calls `aPair.token0/1()`. Seems neater that way too, wdyt?

Reply:
Hmm, I like that this one requires  you to specify the tokens and be very intentional. Would just make the assert a require

Reply:
yup it's very intentional. Made the change, still using if statements instead of require to keep consistent with the overall style